### DISCUSSION TITLE
disk cloning

### DISCUSSION THREAD

**Discussion by CharlNicky** [community/community#122277] [New to GitHub]
create a 5-powerpoint slide demonstration for step by step disk cloning from disk c to disk d

**Comment by ebndev** [comment]
Hi @CharlNicky,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on support for this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
